internal/backend/httpsvc/config.go

```diff
@@ -34,49 +34,60 @@ type CORSConfig struct {
 
 type Config struct {
 	// local server address, set to run server on different port
-	Addr string `koanf:"addr"`
+	Addr    string `koanf:"addr"`
+	AuxAddr string `koanf:"aux_addr"`
```

Comment:
Undocumented, and the reader as no chance to understand what this is/does on their own